lambdas/service/app.py

```diff
@@ -184,20 +184,40 @@ def openapi():
                     body=spec)
 
 
-@app.route('/health', methods=['GET'], cors=True, spec={
+health_spec = {
+    'responses': {
+        '200': {
+            'description': '200 response',
+            'content': {
+                'application/json': {
+                    'schema': {'$ref': '#/components/schemas/Empty'}
+                }
+            }
+        }
+    },
+    'tags': ['Health']
+}
+
+
+@app.route('/health', methods=['GET'], cors=True, method_spec={
+    **health_spec,
+    'summary': 'List health information for webservice'
+})
+@app.route('/health/{keys}', methods=['GET'], cors=True, path_spec={
     'parameters': [
         {
-            'in': 'query',
-            'name': 'foo',
-            'description': 'This is not a real parameter',
-            'required': False,
-            'schema': {'$ref': "#/components/schemas/Empty"}
+            'name': 'keys',
+            'in': 'path',
+            'required': True,
```

Comment:
So despite being required, it's still allowed to be empty? Does `GET /health/` still work?

Reply:
Path parameters are always required https://swagger.io/docs/specification/describing-parameters/#path-parameters. You can still make the request, just not through the Swagger page.